### PR TITLE
Add module product_webcam

### PR DESCRIPTION
**Pull request by misern2**
Taking the hr_webcam module as a reference, I just created a new module called product_webcam in order to take pictures and store them into product table.

### CODE REVIEW

**Comment by misern2** [comment]
I need some help with coveralls check. Where I can get more info ?

**Comment by pedrobaeza** [comment]
Clicking on details you get the code lines that are not tested. As this module is not difficult to test (how do you automatically test a webcam?), it's not mandatory. I personally don't like this one, because it uses a Flash component, but I'm not going to block it.

**Comment by bealdav** [comment]
Thanks for this contribution, but when you say 

> Taking the hr_webcam module as a reference

you mean there is duplicate code ? If yes, there is an opportunity to make a base_ module, don't you think ?

File: product_webcam/product_webcam_data.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data>
+        
+        <record id="action_client_employees_menu" model="ir.actions.client">
+            <field name="name">Open Employees Menu</field>
```
Comment (tremlin) [inline product_webcam/product_webcam_data.xml:6]: There are some copy&paste artifacts from `hr_webcam`.

A proper base module in `web_webcam`(?) for all the common webcam-related web components would probably be the best solution.

**Comment by misern2** [comment]
You can close this PR if you want. I think this module https://github.com/OCA/web/tree/10.0/web_widget_image_webcam is a better solution.

**Pull request closed without merging**